zhaquirks/xiaomi/aqara/ctrl_neutral.py

```diff
@@ -53,6 +62,73 @@
 class CtrlNeutral(XiaomiCustomDevice):
     """Aqara single and double key switch device."""
 
+    class BasicClusterDecoupled(BasicCluster):
+        """Adds attributes for decoupled mode."""
+
+        # Known Options for 'decoupled_mode_<button>':
+        # * 254 (decoupled)
+        # * 18 (relay controlled)
+        manufacturer_attributes = {
+            0xFF22: ("decoupled_mode_left", t.uint8_t),
+            0xFF23: ("decoupled_mode_right", t.uint8_t),
+        }
+
+    class WallSwitchOnOffCluster(OnOff, EventableCluster):
+        """WallSwitchOnOffCluster: fire events corresponding to press type."""
+
+        press_type = {
+            0x00: SHORT_PRESS,
+            0x01: SHORT_RELEASE,
+            0x02: DOUBLE_PRESS,
+        }
+        name = "WallSwitch_cluster"
+        ep_attribute = "WallSwitch_cluster"
+
+        def __init__(self, *args, **kwargs):
+            """Init."""
+            self.last_tsn = -1
+            super().__init__(*args, **kwargs)
```

Comment:
You don't need this. Not unless it sends two consecutive commands with the same TSN, which was only tuya specific thing